### PR TITLE
Add some NA settings

### PR DESCRIPTION
**Pull request by ApeMV**

### CODE REVIEW

**Comment by Pietrodjaowjao** [comment]
You have been banned for hacking

Please stop.

**Comment by NikoCat233** [comment]
It is suggested that you dont change settings id for existing options
Pls do me a favor next time =(

File: Roles/Neutral/Berserker.cs [hunk 1]
```diff
@@ -161,7 +164,7 @@ public override bool OnCheckMurderAsKiller(PlayerControl killer, PlayerControl t
                 if (player == killer) continue;
                 if (player == target) continue;
 
-                if (Utils.GetDistance(killer.transform.position, player.transform.position) <= Bomber.BomberRadius.GetFloat())
```
Comment (NikoCat233) [inline Roles/Neutral/Berserker.cs:164]: What does this do?

**Review by NikoCat233 — CHANGES_REQUESTED** [review]

File: Roles/Neutral/Berserker.cs (hunk 1 above)
Comment (ApeMV) [reply Roles/Neutral/Berserker.cs:164]: I added a check for if can kill teamate is true or false.

**Comment by ApeMV** [comment]
Also if one more person is going to mention the ID's I am gonna go crazy lmao, I know not to change them now.

File: Roles/Neutral/Berserker.cs (hunk 1 above)
Comment (ApeMV) [reply Roles/Neutral/Berserker.cs:164]: Nvm this code is foul

Comment (ApeMV) [reply Roles/Neutral/Berserker.cs:164]: Fixed, idk what I was thinking making this lol

**Review by NikoCat233 — APPROVED** [review]

**Pull request merged**